test/general/packet.js

```diff
@@ -972,4 +972,102 @@ V+HOQJQxXJkVRYa3QrFUehiMzTeqqMdgC6ZqJy7+
       ).to.be.rejectedWith(/User ID string is too long/);
     });
   });
+
+  it('Public key encrypted symmetric key packet using symmetric keys', () => {
+    const serializedPacket = new Uint8Array([
+      0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x64, 0x01, 0x4c, 0x80, 0x3d, 0x9b, 0xb6,
+      0x44, 0x13, 0x25, 0x90, 0x24, 0x6e, 0x59, 0x0d,
+      0x68, 0xee, 0x1e, 0x23, 0x75, 0x8a, 0x90,
+      0x85, 0x78, 0x1b, 0xf7, 0xc4, 0x4d, 0x58, 0xc7,
```

Comment:
This is very nitpicky but maybe reformat this packet? It looks a bit strange now :sweat_smile:

Reply:
Yeah I thought about this. I ended up leaving it as is since it keeps the data aligned the same way as in `expectedCipherText`. But yeah I guess it looks too weird.

Reply:
Oh, yeah, I didn't notice that. I guess that one was copied from this one, so if you want, you could reformat that one too :shrug: